lib/lotus_notes/person_event.rb

```diff
@@ -0,0 +1,83 @@
+module LotusNotes
+  class PersonEvent
```

Comment:
This class does two things:

1 - wraps the results of the request into an object
2 - syncs with PersonEvent

Given that this is a lib class, it should be agnostic about PersonEvent, so my suggestion is remove the synchronization part from here and move it to another service class or to `LotusNotesCalendarIntegration`